iroh-gateway/src/rpc.rs

```diff
@@ -1,6 +1,6 @@
 use anyhow::Result;
 use async_trait::async_trait;
-use iroh_rpc_types::gateway::{Gateway as RpcGateway, GatewayServerAddr, VersionResponse};
+use iroh_rpc_types::gateway::{GatewayGateway as RpcGateway, GatewayServerAddr, VersionResponse};
```

Comment:
Can we fix these not to be `GatewayGateway` or `P2pP2p` etc ?

Reply:
Maybe, but it all comes from the protobuf grpc structure.